### ISSUE TITLE
Is it possible to have categories on the left nav?

### ISSUE THREAD

**Issue by oren** [rust-lang-nursery/mdBook#932]
I was able to remove the numbers using `no-section-label = true` but I want to create categories that have different style that the pages they contain. Here is an example: https://oren.github.io/surf

**Comment by krisradio** [comment]
+1, would be very cool, might be out of scope of the project tho.

**Comment by evitalis** [comment]
I don't think this should be out of scope as it makes more sense than numbers for many cases. Just discovered this though and support this idea.

**Comment by alexwanng** [comment]
+1. especially when having lots of pages.
